src/error/specification

```diff
@@ -24,19 +24,19 @@ number:C01110
 description:Out of Memory
 macro:OUT_OF_MEMORY
 
-number:C01300
+number:C01200
```

Comment:
Didn't notice it earlier but: is this a "number"? Shouldn't it by an id/identifier?

Reply:
It is an id. The initial Error codes though changed a lot since the beginning of the big PR and i forgot to synchronize it with the design decision once. Since every code is integrated in a macro it is quite easy to change :)